NSS-Day-Cohort-42/rare-API-media-moguls
# Action added to posts view

#### Changes Made
1. Added an action to the post.py file in views. 
2. Added one more post fixture to posts.json for testing purposes
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout sm-postapproval
```

2. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
5. Test app functionality.
> Make sure to run `python manage.py loaddata posts` in your terminal to ensure that you have the new fixture.
>In Postman run a get for all posts to make sure that the extra fixture was added to your database(it should have `approved = False` )
>Change the method to PUT on postman and enter the following URL `http://localhost:8000/posts/4/approval`, You should see a 204 no content code
>Get all posts again and see that approved on that specific object is now equal to true!

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
